datafusion/functions-aggregate/COMMENTS.md

```diff
@@ -54,7 +54,7 @@ first argument and the definition looks like this:
 // `input_type` :  data type of the first argument
 let mut fields = vec![Field::new_list(
     format_state_name(self.name(), "nth_value"),
-    Field::new("item", args.input_type.clone(), true /* nullable of list item */ ),
+    Field::new("item", args.input_types.clone(), true /* nullable of list item */ ),
```

Comment:
Thanks @alamb for doing this 🙏.

The second argument should be `args.input_types[0].clone` according to the changes in this PR.

Reply:
```suggestion
    Field::new("item", args.input_types[0].clone(), true /* nullable of list item */ ),
```